types/astring/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+		"no-padding": false,
```

Comment:
Please remove all the rules from here. We want only extends clause in new definitions

Reply:
before doing so I have two questions:
1. Without these settings, using the automatic formatting in VS Code will produce lint errors. Is there a config file I could use? I think automatic formatting should be used for the sake of readability and consistency.
2. When I run "npm test" locally, I never got any errors, but the CI build did (see https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/442690632). It seems like the extending of dt.json does not work correctly locally. Am I missing something or should I open a bug report?